libraries/messages/src/lib.rs

```diff
@@ -47,6 +50,7 @@ pub struct Message {
 pub enum Data {
```

Comment:
This is failing; "the trait bound `Log: TS` is not satisfied".

Reply:
Since those logs are never sent to ground station, I've decided to remove the derive for that enum.